src/App.php

```diff
@@ -547,7 +617,7 @@ public function runFrontend(App\Module $module, App\Router $router, IPConfig $pc
 				$this->baseURL->redirect('search');
 			}
 
-			// Initialize module that can set the current theme in the init() method, either directly or via App->profile_uid
+			// Initialize module that can set the current theme in the init() method, either directly or via App->profile_owner
```

Comment:
Shouldn't we mention the setter method instead?

Reply:
Possibly.